contrib/src/main/java/com/datatorrent/contrib/hbase/AbstractHBaseWindowAppendOutputOperator.java

```diff
@@ -46,8 +40,6 @@
  * for the tuple in the table.<br>
  *
  * <br>
- * This class provides batch append where tuples are collected till the end
- * window and they are appended on end window
```

Comment:
With this change, the operator is no different from `AbstractHBaseAppendOutputOperator`. So:

- why can't we support batch append even with multi-table support? Pls see comments below.
- Isn't the class name `...WindowAppend...` misleading if there is no batch or window append support? Can we make it deprecated and prevent creation of this operator?